DOC: Plotting / Visualization without Jupyter (#47948)
pandas-dev/pandas: doc/source/user_guide/visualization.rst

Chart visualization
*******************

.. note::

>    The examples below assume that you're using `jupyter <https://jupyter.org/>`_.

Reviewer: ```suggestion
   The examples below assume that you're using `Jupyter <https://jupyter.org/>`_.
```

> 

Reviewer 2: suggestion, add: 
```
    If one is implementing a script or is executing the Python interpreter
    from a command line interface, one may opt for ``plt.savefig('$filename')``
    in lieu of the ``$object.plot()`` functions found in this tutorial.

    ``plt.savefig()`` saves the pandas object as an image and accepts a variety  
    of filetypes, including png, pdf, jpg, and svg. 
```

Reviewer: This isn't quite right, there's a button to press to suggest changes, check the article I linked (step 6)

Reviewer 3: IMO I don't think this note is entirely necessary. Since the rest of the documentation mentions using the matplotlib API, I think further matplotlib actions can be independently looked-up by the user.

Reviewer: That's a good point - let's keep as-is then

>    And the examples below assume that `jupyter <https://jupyter.org/>`_ is being used.

Reviewer: Better to not start a sentence with "and"
```suggestion
   The examples below assume that you're using `jupyter <https://jupyter.org/>`_.
```
